### ISSUE TITLE
if i want The training input data is four channels, because my six-channel data is spliced ​​from six single-channel 2D images. How should I modify the code? I do not have a mask annotation file. Can I use this code?

### ISSUE THREAD

**Issue by watertianyi** [TheStarOfMSY/MoEAD#2]

**Comment by TheStarOfMSY** [comment]
Thank you for your question. This network uses pre-trained EfficientNet-b4, which is for 3 channels of input data. So you may change the backbone and choose a new pre-trained model to get the features. The mask annotation file is used for the calculation of accuracy metrics. You can still train the model, but the evaluation may have problems.

**Comment by watertianyi** [comment]
@TheStarOfMSY 

Hello author, if I only use normal images for training and real abnormal images for reasoning, I hope to get the defect heat map in the abnormal images and the intensity of the defects can be clearly distinguished. Is this code suitable?

**Comment by TheStarOfMSY** [comment]
Yes, the model is suitable. But some changes in code are necessary.
